### PR TITLE
tutorial revision

### PR DESCRIPTION
**Pull request by nksaraf**

### CODE REVIEW

**Comment by Jutanium** [comment]
This PR now has the major infra updates and the most important revisions to the tutorial that make it easier to follow along.

File: content/start/core-concepts/resource-routes.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- Creating a PATCH endpoint
+- Creating a `users.json` endpoint
+
+# Resource Routes
+
+Resource Routes are not part of your application UI, but are still part of your application. They can send any kind of Response.
+
+Most routes in SolidStart are UI Routes, or routes that actually render a component. But routes don't always have to render components. There are a handful of cases where you want to use route as a general purpose endpoint to your website. Here are a few examples:
+
+- JSON API for a mobile app that reuses server-side code with the Remix UI
```
Comment (erikgaal) [inline content/start/core-concepts/resource-routes.mdx:16]: Remix reference.

File: content/start/core-concepts/resource-routes.mdx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+};
+```
+
+## Webhooks
+
+Resource routes can be used to handle webhooks. For example, you can create a webhook that receives notifications from GitHub when a new commit is pushed to a repository:
+
+```ts
+import type { ActionFunction } from "@remix-run/node"; // or cloudflare/deno
+import { json } from "@remix-run/node"; // or cloudflare/deno
```
Comment (erikgaal) [inline content/start/core-concepts/resource-routes.mdx:111]: Remix reference.

**Review by erikgaal — COMMENTED** [review]
Looks like this PR introduced multiple verbatim copies of the Remix docs, clearly noticeable by the references to it in the source.

Ex. https://github.com/remix-run/remix/blob/main/docs/guides/resource-routes.md

[The docs of Remix are licensed under MIT](https://github.com/remix-run/remix/blob/main/LICENSE.md) which means that you need to AT LEAST provide license and copyright notice. Under the current state it is in violation of that license.

**Comment by Jutanium** [comment]
@erikgaal To be clear, the SolidStart docs are not meant to be published or linked from anywhere. I believe @nksaraf used this as placeholder text; regardless, I'll be removing it.

**Comment by erikgaal** [comment]
Good to hear @Jutanium. I understand your reasoning, however, it's publicly accessible on a _major_ development platform which could be considered a publication by itself already.

**Comment by ryanflorence** [comment]
> used this as placeholder text

🙄 It's got enough solid specific stuff around our content that it's clearly a purposeful plagiarism. You can run the diff yourself to see. Lorem ipsum is placeholder text.

Our docs are MIT licensed, so you're actually violating the license right now. If we audit the source code of SolidStart, will we find similar copyright violations?

**Comment by phorcys420** [comment]
just get it over with instead of attacking eachother, they probably wanted to test out a featureful page that actually has markdown

**Comment by ryansolid** [comment]
> If we audit the source code of SolidStart, will we find similar copyright violations?

We are crediting Remix where we are using portions of source code. Things like forms and cookies. I will perform an audit to make sure there aren't any violations.

Ryan, we apologize for this mixup. SolidStart is still pre-release. The Remix docs were introduced as a placeholder to prove out the Infrastructure and weren’t intended to be public. We are taking action to address this.

**Comment by space-otter** [comment]
@ryanflorence It would also be kind of you if you give @ryansolid, @Jutanium or somebody else an opportunity to react on your Tweet. I don't think it is in anybody's interest to let people hop on a toxic wagon unloaded with context.

**Comment by narutosstudent** [comment]
Agreed with @space-otter, IMO with all due respect 🙏  this could've been handled differently by talking to @ryansolid and @Jutanium privately, instead of publicly tweeting it.

@ryanflorence You closed the replies but not the fact that others can quote tweet it, which essentially spread the word and looking at the quote tweets some of them are assuming far negative than the actual situation 🤷‍♂️ .

As someone who loves both Remix and Solid.js, I wish this was handled differently, much love! ❤️ .

I don't know the full context, I'm not assuming anything, I'm just wishing it was done privately between yall since it is sort of hurting the name of Solid.js and the people behind/around it.

Again, much love  ❤️

**Pull request merged**